code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -399,3 +399,47 @@
 /obj/item/gun/syringe/rapidsyringe/preloaded/beaker_blaster/attack_self(mob/living/user)
 	// no printing infinite syringes.
 	return
+
+/// Version for Malf Borgs
+/obj/item/gun/syringe/malf
+	name = "plasma syringe cannon"
+	desc = "A syringe gun integrated into a medical cyborg's chassis. Fires heavy-duty plasma syringes tipped in poison."
+	icon_state = "rapidsyringegun"
+	max_syringes = 14
+
+//Preload Syringes
+/obj/item/gun/syringe/malf/Initialize(mapload)
+	..()
+	while(length(syringes) + (chambered.BB ? 1 : 0) < max_syringes)
+		var/obj/item/reagent_containers/syringe/S = new /obj/item/reagent_containers/syringe
```

Comment:
Should these be heavy duty syringes?

Reply:
The heavy duty syringe is a projectile, not necessarily a reagent container. It operates in the same way the RSG turns a syringe into a piercing syringe when fired.